aspnetcore/migration/mvc2.md

```diff
@@ -0,0 +1,122 @@
+
+In .Net Framework Applications, developers often rely on an external package to configure Dependecy Injection, one notable package being [Unity](https://github.com/unitycontainer/unity), provided by Microsoft Patterns & Practices. 
+
+One example of setting up Dependency Injection with Unity is implementing IDependencyResolver that wraps a Unity Container
+
+[!code-csharp[Main](../../aspnet/web-api/overview/advanced/dependency-injection/samples/sample8.cs)]
+
+Then you create an instance of your UnityContainer, register your service and set the dependency resolver of HttpConfiguration to new instance of UnityResolver for your container
+
+[!code-csharp[Main](../../aspnet/web-api/overview/advanced/dependency-injection/samples/sample9.cs)]
+
+Now you can inject IProductRepository anywhere you need to
+
+[!code-csharp[Main](../../aspnet/web-api/overview/advanced/dependency-injection/samples/sample5.cs)]
+
+It isn't much work, but it is work regardless. Now that Dependency Injection is included with .Net Core, all that is required is adding your service in the `ConfigureServices` section of `Startup.cs`
+
+[!code-csharp[Main](samples/sample.cs)]
+
+That is it, you now can inject your repository anywhere like before. This process is now streamlined and reduces complexity as well as dependencies on external packages.
+> [!NOTE]
+> For a more in-depth reference to .Net Core and Dependecy Injection, please read Steve Smith and Scott Addie's article on Dependecy [Injection in ASP.Net Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection#replacing-the-default-services-container)
+
+## Static file / wwwroot
+One essential part of web development is the ability to serve files that are "static" or in better terms, not built code to the browser. The most common examples of static files are HTML, CSS, Javascript and images and they are used by every website on the internet. We need to be able to save these files in the published location of our application (or CDN) and reference these files to load onto the browser. This process has changed in Dot Net Core wheras in previous versions of the .Net Framework, you could store your static files basically anywhere, and reference them in your view like regular html (with the path to the asset provided). 
+
+In Dot Net Core, we store our static files in `web root` (<content root>/wwwroot) and loading them into the pipeline using the `UseStaticFiles` extension from `Statup.Configure`. Be sure to include the NuGet package "Microsoft.AspNetCore.StaticFiles" in your application.
```

Comment:
- Dot Net Core --> .NET Core
- remove the back ticks surrounding "web root"
- "Microsoft.AspNetCore.StaticFiles" --> `Microsoft.AspNetCore.StaticFiles`